src/index.html

```diff
@@ -1,17 +1,133 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+  <!-- action="https://mate-academy-form-lesson.herokuapp.com/create-application" -->
+
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <div class="container">
+      <form action="smth.js" method="POST"
+      onsubmit="onSubmit()">
+        <fieldset>
+          <legend>Personal information</legend>
+          <p>
+            <label for="surname">Surname:
+              <input id="surname" name="surname" 
```

Comment:
Починай текст тег з нового рядка
```suggestion
            <label for="surname">
              Surname:
              <input id="surname" name="surname" 
```